go.sum

```diff
@@ -36,13 +36,13 @@ github.com/contiamo/go-base/v2 v2.4.2/go.mod h1:SU3nkAtieXvhZl/4RdrgQIhzddAkKAYr
 github.com/contiamo/goserver v0.5.0/go.mod h1:OyvuGbzBny2rw3LUT5x3TZojag6+euu1CbhwG63+0rQ=
 github.com/contiamo/goserver v0.5.2 h1:r6p4lF/an/JkVG8MGGOtU8g6pQlb3AtjYbpEJiD3oXo=
 github.com/contiamo/goserver v0.5.2/go.mod h1:PRq+h0iQLYJhKUmZutevkA0B8ftPE0chwPNNoT9P+g0=
-github.com/contiamo/jwt v0.2.2 h1:l/GTqHdd8coYqLBfDbImlDcwqh0sJyaAru20ggfMofE=
 github.com/contiamo/jwt v0.2.2/go.mod h1:7dNxJJt3avZrEoVk4XUJbHuL6ARCmw7o+GAfx9IW/Ik=
+github.com/contiamo/jwt v0.3.0 h1:82BYt88a2oUYjptewxIzw9TkbQjnNqZAQqigWc6tYvw=
+github.com/contiamo/jwt v0.3.0/go.mod h1:PyKnz0U4z7NfpvmzuHTCMIocnmbaaWxtDhJdvn5nuCM=
 github.com/creack/pty v1.1.9/go.mod h1:oKZEueFk5CKHvIhNR5MUki03XCEU+Q6VDXinZuGJ33E=
 github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
 github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c=
 github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
-github.com/dgrijalva/jwt-go v3.2.0+incompatible h1:7qlOGliEKZXTDg6OTjfoBKDXWrumCAMpl/TFQ4/5kLM=
 github.com/dgrijalva/jwt-go v3.2.0+incompatible/go.mod h1:E3ru+11k8xSBh+hMPgOLZmtrrCbhqsmaPHjLKYnJCaQ=
```

Comment:
This is still left because of this https://github.com/contiamo/goserver/blob/798094efecde8127a129a0237e8a9ab192d723f0/go.mod#L7